src/test/apps.test.js

```diff
@@ -563,7 +563,7 @@ describe("App validator", function () {
 describe("App Page component", function () {
 
   beforeEach(function () {
-    var app = _.extend(appScheme, {
+    var app = React.addons.update(appScheme, {$merge: {
```

Comment:
Mmmh... I guess we could but then util.js will always depend on React.addons (well, I guess React is a hard dependency anyway...)